test/metabase/test_setup.clj

```diff
@@ -99,3 +99,17 @@
   []
   (log/info "Shutting down Metabase unit test runner")
   (core/stop-jetty!))
+
+(defn call-with-test-scaffolding
```

Comment:
It's only used at the REPL so should be fine to mark as private, I can just invoke it via the var